Makefile

```
@@ -31,7 +31,9 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 BUNDLE_IMG ?= controller-bundle:$(VERSION)
 
 # Image URL to use all building/pushing image targets
-IMG ?= <operator-registry>/mongodb-kubernetes-operator
+REPO_URL := $(shell cat ~/.community-operator-dev/config.json | jq -r .repo_url)
+OPERATOR_IMAGE := $(shell cat ~/.community-operator-dev/config.json | jq -r .operator_image)
```

Maybe do this if you think it is more readable (I do)

```suggestion
REPO_URL := $(shell jq -r .repo_url ~/.community-operator-dev/config.json)
OPERATOR_IMAGE := $(shell jq -r .operator_image ~/.community-operator-dev/config.json)
```

---

I had initially tried this, but for some reason the variables were not being populated properly. Let me try it one more time and if it works I will incorporate these changes!